### DISCUSSION TITLE
(Diamond) There are screen shifts.

### DISCUSSION THREAD

**Discussion by evrenonur** [primefaces/community#1168] [PrimeVue]
In the Diamond Vue version, there is a collaboration with git. When the next person cloned the project on his/her computer, it was observed that the screens shifted and some areas did not work
![Ekran görüntüsü 2024-02-10 092520](https://github.com/primefaces/community/assets/49561437/a8cae6d7-6ca6-4e54-ab36-40ea0d4ae3d2)
![Ekran görüntüsü 2024-02-10 092534](https://github.com/primefaces/community/assets/49561437/8b2874be-8a58-47dd-bd32-cbfb351045ec)
![Ekran görüntüsü 2024-02-10 092546](https://github.com/primefaces/community/assets/49561437/8012dc2b-1d9d-46a9-81b0-634d21726be6)

**Comment by tugcekucukoglu** [comment]
Which version are you using for Diamond and PrimeVue?

**Reply by evrenonur** [reply]
I got the errors I wrote when I used Diamond Vue version 5.2. But in version 5.1, it does not get errors now, but I want it to be up to date

**Reply by gongora92** [reply]
I got the same error. I am using Diamond 5.2 and primevue 3.36
